CHANGELOG.md

```diff
@@ -1,5 +1,121 @@
 # Changelog
 
+# 25.0 [#781](https://github.com/openfisca/openfisca-core/pull/781)
+
+#### Breaking changes
+
+  - Change the syntax of OpenFisca YAML tests
+
+  For instance, a test that was using the `input_variable` keyword like:
+
+  ```yaml
+  - name: Basic income
+    period: 2016-12
+    input_variables:
+      salary: 1200
+    output_variables:
+      basic_income: 600
+  ```
+
+  becomes:
+
+  ```yaml
+  - name: Basic income
+    period: 2016-12
+    input:
+      salary: 1200
+    output:
+      basic_income: 600
+  ```
+
+  A test that was fully specifying its entities like:
+
+  ```yaml
+  name: Housing tax
+    period: 2017-01
+    households:
+      - parents: [ Alicia ]
+        children: [ Michael ]
+    persons:
+      - id: Alicia
+          birth: 1961-01-15
+      - id: Michael
+          birth: 2002-01-15
+    output_variables:
+      housing_tax:
+        2017: 1000
+  ```
+
+  becomes:
+
+  ```yaml
+  name: Housing tax
+    period: 2017-01
+    input:
+      household:
+        parents: [ Alicia ]
+        children: [ Michael ]
+      persons:
+        Alicia:
+          birth: 1961-01-15
+        Michael:
+          birth: 2002-01-15
+    output:
+      housing_tax:
+        2017: 1000
+  ```
+
+  A **migration script** is available to automatically convert tests:
+
+  ```sh
+    python openfisca_core/scripts/migrations/v24_to_25.py /path/to/tests/
+  ```
+
+  > Note for country package using Scenarios (e.g. France, Tunisia):
+  > Tests are not using scenarios anymore. Therefore, tests cannot partially specify entities anymore. Tests using entities inference may need manual adaptation in addition to the script.
+
+  - The `Entity` constructor (usually not directly called by users) does not accept an `entities_json` parameter anymore.
+
+
+#### Deprecation
+
+  - Deprecate `openfisca-run-test`
+    - `openfisca test` should be used instead.
```

Comment:
Unifying `openfisca serve` and `openfisca-run-test` in a single `openfisca` command was the plan when `openfisca serve` was introduced.

This seem like a good time to do it. It could be `openfisca run-test` if you think it's better, no strong opinion (but `test` is shorter and also a verb.)

Reply:
`openfisca test`!